packages/janus-js/janus-js.dev/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+maintainer: "sashayanin@gmail.com"
+authors: "Alexander Yanin"
+homepage: "https://github.com/monstasat/janus-ocaml"
+dev-repo: "git+https://github.com/monstasat/janus-ocaml.git"
+bug-reports: "https://github.com/monstasat/janus-ocaml/issues"
+license: "MIT"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore, your dune-project file says you require at least dune 1.5